projects/www/src/app/examples/store-walkthrough/src/book-collection/book-collection.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Book } from '../book-list/books.model';
+
+@Component({
+  selector: 'app-book-collection',
+  template: `
+    @for(book of books; track book) {
+    <div class="book-item">
+      <p>{{ book.volumeInfo.title }}</p>
+      <span> by {{ book.volumeInfo.authors }}</span>
+      <button (click)="remove.emit(book.id)">Remove from Collection</button>
+    </div>
+    }
+  `,
+})
+export class BookCollection {
+  @Input()
```

Comment:
For some reason the example doesn't run on stackblitz with the signal input and new output API.